app/src/main/java/com/mapbox/maps/testapp/examples/terrain3D/ModelLayerActivity.kt

```diff
@@ -0,0 +1,78 @@
+package com.mapbox.maps.testapp.examples.terrain3D
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.mapbox.geojson.Feature
+import com.mapbox.geojson.FeatureCollection
+import com.mapbox.geojson.Point
+import com.mapbox.maps.MapView
+import com.mapbox.maps.Style
+import com.mapbox.maps.dsl.cameraOptions
+import com.mapbox.maps.extension.style.expressions.dsl.generated.get
+import com.mapbox.maps.extension.style.layers.generated.modelLayer
+import com.mapbox.maps.extension.style.layers.properties.generated.ModelType
+import com.mapbox.maps.extension.style.model.model
+import com.mapbox.maps.extension.style.sources.generated.geoJsonSource
+import com.mapbox.maps.extension.style.style
+import com.mapbox.turf.TurfMeasurement
+
+/**
+ * Showcase adding a 3D model using model layer.
+ */
+class ModelLayerActivity : AppCompatActivity() {
```

Comment:
do we also want to add MapboxExperimental annotation here? I recall doing that in some other places but guess it's not the rule